# <img src="https://coursesandconferences.wellcomeconnectingscience.org/wp-content/themes/wcc_courses_and_conferences/dist/assets/svg/logo.svg" width="300" height="50"> 
# Next Generation Sequencing

# 24–31 March 2023, Wellcome Genome Campus, UK

[Next Generation Sequencing Course Website Link](https://coursesandconferences.wellcomeconnectingscience.org/event/next-generation-sequencing-20230324/) <br /> 

# Summary

Next generation sequencing (NGS) has become the premier tool in genetic and genomic analysis. This hands-on laboratory and informatics  course is directed at scientists who will be generating and interpreting sequence information in their research and wish to improve their understanding of various sequencing platforms, techniques,  and their applications.

Participants will be able to make informed decisions about appropriate techniques to apply in a variety of research contexts. Training will cover applications for target enrichment, bacterial sequencing, cancer genomic analysis, human variation analysis, and RNA-Seq.

The programme will include theoretical and practical information on NGS systems currently available and those on the near horizon. Technologies will be chosen from short- and long-read sequencing platforms including:

- PacBio
- Illumina Miseq/HiSeq/Novaseq
- Oxford Nanopore MinION

All the basic techniques of post-sequencing analysis will also be covered in hands-on genomics data analysis sessions including QC, sequence alignment, and variant calling.

# Target Audience

Applicants should be postdoctoral scientists, senior PhD students, or junior faculty members actively engaged in or soon to commence research involving next generation sequencing instrumentation.

**Prerequisites**

Please note: For the data analysis components of the course, applicants will require a working knowledge of the UNIX/Linux operating system. This may be obtained through various methods and is essential to fully benefit from the course. There are numerous online introductory tutorials to the UNIX/Linux operating system and command line, including:

- [http://www.ee.surrey.ac.uk/Teaching/Unix](http://www.ee.surrey.ac.uk/Teaching/Unix)
- [http://swcarpentry.github.io/shell-novice/](http://swcarpentry.github.io/shell-novice/)

This course is not meant to replace the manufacturers’ training normally supplied with new instruments, nor is it intended to be a training course for those solely interested in NGS data analysis (please refer to our website for dedicated bioinformatics courses).

## Learning outcomes

Upon completion of the course, participants should be able to:

- prepare high quality sample libraries for use on next generation sequencing platforms
- set up sequencing runs for the various platforms utilised during this course
- critically assess a sequencing run and distinguish between good and poor-quality data
- describe the advantages and disadvantages of different sequencing technologies in order to select technologies appropriate to specific needs
- process NGS data and use computational tools for sequence alignment and variant-calling

# Programme

**The course will run approximately 08:30-18:00 (BST) daily.** 

The programme will include lectures, seminars, practical sessions, and social events covering the following topics:

**Library Prep**

- Preparing high-quality libraries from samples
- Sequencing

**Overview: Motivation and fundamental concepts**

- Detailed description of prominent sequencing technologies: current and emerging platforms
- preparing chips for sequencing and performing sequencing runs on various platforms
Data analysis

**Data QC: How to determine if a sequencing run has performed well**

- Alignment to a reference sequence and variant-calling of NGS data
- The course will also include seminars by invited speakers, who will highlight their ground-breaking work in the applications of next generation sequencing.

## Computational Practicals

- [Unix for Bioinformatics Slides](https://github.com/WCSCourses/NGS23/blob/main/Modules/Unix_for_Bioinformatics/linux_slides.pdf)
- [Unix for Bioinformatics Practical](https://github.com/WCSCourses/NGS23/blob/main/Modules/Unix_for_Bioinformatics/linux_practical.pdf)
- [NGS Data Formats Slides](https://github.com/WCSCourses/NGS23/blob/main/Modules/NGS_Data_Formats_and_QC/data_formats_slides.pdf)
- [QC Slides](https://github.com/WCSCourses/NGS23/blob/main/Modules/NGS_Data_Formats_and_QC/qc_slides.pdf)
- [NGS Data Formats & QC Slides](https://github.com/WCSCourses/NGS23/blob/main/Modules/NGS_Data_Formats_and_QC/data_formats_practical.pdf)
- [Read Alignment Slides](https://github.com/WCSCourses/NGS23/blob/main/Modules/Read_Alignment/NGS-2023-Alignment.pdf)
- [Read Alignment Practical](https://github.com/WCSCourses/NGS23/blob/main/Modules/Read_Alignment/read_alignment_practical.pdf)
- [Variant Calling Slides](https://github.com/WCSCourses/NGS23/blob/main/Modules/Variant_Calling/variant-calling-talk.pdf)
- [Variant Calling Practical](https://github.com/WCSCourses/NGS23/blob/main/Modules/Variant_Calling/variant_calling_practical.pdf)
- [Long Read Analysis Slides](https://github.com/WCSCourses/NGS23/blob/main/Modules/Long_Read_Analysis/Oxford_Nanopore_2023_plus_analysis.pdf)
- [Long Read Analysis Practical as PDF](https://github.com/WCSCourses/NGS23/blob/main/Modules/Long_Read_Analysis/WCS%20Intro%20to%20NGS%20Oxford%20Nanopore%20Analysis%20session.pdf)
- [Long Read Analysis Practical as Jupyter Notebook](https://github.com/WCSCourses/NGS23/blob/main/Modules/Long_Read_Analysis/WCS%20Intro%20to%20NGS%20Oxford%20Nanopore%20Analysis%20session.ipynb)

**Course Instructors**      

- [Jacqueline Keane](https://www.sanger.ac.uk/person/keane-jacqueline/)
- [Kim Judge](https://www.sanger.ac.uk/person/judge-kim/)
- [Michael Quail](https://www.sanger.ac.uk/person/quail-michael-andrew/)

******

## Citing and Re-using Course Material

The course data are free to reuse and adapt with appropriate attribution. All course data in these repositories are licensed under the <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</a>. <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><br /> 

## Interested in attending a course?

Take a look what courses are coming up at [Wellcome Connecting Science Courses & Conference Website](https://coursesandconferences.wellcomeconnectingscience.org/our-events/).

---

For more information or queries, feel free to contact us via the [Wellcome Connecting Science website](https://coursesandconferences.wellcomeconnectingscience.org).<br /> 
Find us on socials [Wellcome Connecting Science Linktr](https://linktr.ee/eventswcs)

---

[Wellcome Connecting Science GitHub Home Page](https://github.com/WCSCourses) <br /> 
